app/src/main/kotlin/com/simplemobiletools/gallery/pro/dialogs/PickDirectoryDialog.kt

```diff
@@ -32,6 +34,9 @@ class PickDirectoryDialog(
     private var isGridViewType = activity.config.viewTypeFolders == VIEW_TYPE_GRID
     private var showHidden = activity.config.shouldShowHidden
     private var currentPathPrefix = ""
+    private val config = view.context.config
+    private val searchView = view.folder_search_view
+    private val searchEditText = view.findViewById<EditText>(R.id.top_toolbar_search)
```

Comment:
`view` is  an instance of `MySearchMenu` which is placed at `Simple-Commons`.
not a good practice to access an internal of custom view.